src/gcp/cloudfunctions.ts

```diff
@@ -457,7 +463,12 @@ export async function listFunctions(projectId: string, region: string): Promise<
  * List all existing Cloud Functions in a project.
  * @param projectId the Id of the project to check.
  */
-export async function listAllFunctions(projectId: string): Promise<CloudFunction[]> {
+export async function listAllFunctions(
+  projectId: string
+): Promise<{
```

Comment:
Nit: consider making this a named type so tests can declare things to use that type.